satellite/console/consoleweb/consoleapi/payments.go

```diff
@@ -0,0 +1,104 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package consoleapi
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"go.uber.org/zap"
+	monkit "gopkg.in/spacemonkeygo/monkit.v2"
+
+	"storj.io/storj/satellite/console"
+)
+
+var mon = monkit.Package()
+
+// Payments is an api controller that exposes all payment related functionality
+type Payments struct {
+	log     *zap.Logger
+	service *console.Service
+}
+
+// NewPayments is a constructor for api payments controller.
+func NewPayments(log *zap.Logger, service *console.Service) *Payments {
+	return &Payments{
+		log:     log,
+		service: service,
+	}
+}
+
+// SetupAccount creates a payment account for the user.
+func (p *Payments) SetupAccount(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	var err error
+	defer mon.Task()(&ctx)(&err)
+
+	err = p.service.Payments().SetupAccount(ctx)
+	if err != nil {
+		p.serveJSONError(w, http.StatusInternalServerError, err)
+	}
+}
+
+// AccountBalance returns an integer amount in cents that represents the current balance of payment account.
+func (p *Payments) AccountBalance(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	var err error
+	defer mon.Task()(&ctx)(&err)
+
+	balance, err := p.service.Payments().AccountBalance(ctx)
+	if err != nil {
+		p.serveJSONError(w, http.StatusInternalServerError, err)
+	}
+
+	var balanceResponse struct {
+		Balance int64 `json:"balance"`
+	}
+
+	balanceResponse.Balance = balance
+
+	err = json.NewEncoder(w).Encode(balanceResponse)
+	if err != nil {
+		p.serveJSONError(w, http.StatusInternalServerError, err)
```

Comment:
Since the json NewEncoder already started writing something, it will double write the header.